plugin/src/main/java/com/craftmend/openaudiomc/generic/redis/packets/interfaces/OARedisPacket.java

```diff
@@ -12,6 +12,7 @@
     public abstract String serialize();
 
     public OARedisPacket deSerialize(String json) {
+
         return OpenAudioMc.getGson().fromJson(json, getClass());
```

Comment:
When it's a `ExecuteBulkCommandsPacket` class, `getClass()` returns `ExecuteBulkCommandsPacket` and not `OARedisPacket`. 

And `RedisTypeAdapter` is registered for `OARedisPacket`, but not for `ExecuteBulkCommandsPacket`. So here it does not uses the custom `RedisTypeAdapter` but `com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter` instead.